### PR TITLE
Feature/metadataformat

### LINKED ISSUE #1: Define Metadata format for Rego policies
In order to make policies searchable we need to have some form of metadata that maintains relevant search information. We can base this format on e.g. [ArtificatHub](https://artifacthub.io/packages/opa/deprek8ion/deprek8ion) or [Helm Hub](https://hub.helm.sh/charts/fyipe/Fyipe).

At the very minimum the following fields should be included:

- Maintainer(s)
- Version
- Homepage/Link
- Keywords
- Description

Ideally we also support an Icon & a Long README style description.

### PR DESCRIPTION
**Pull request by xchapter7x**
This adds a model and supporting Load function to allow for reading metadata files.

This applies to the issue here https://github.com/policy-hub/policy-hub-cli/issues/1

### CODE REVIEW

File: pkg/metaconfig/metaconfig.go [hunk 1]
```diff
@@ -0,0 +1,20 @@
+package metaconfig
+
+import "gopkg.in/yaml.v2"
+
+type MetadataConfig struct {
```
Comment (Blokje5) [inline pkg/metaconfig/metaconfig.go]: I would argue for just Metadata. MetadataConfig seems to imply this is configuration of Metadata while it is actually just Metadata.

**Review by Blokje5 — CHANGES_REQUESTED** [review]
Small naming request change, but LGTM!

File: pkg/metaconfig/metaconfig.go (hunk 1 above)
Comment (xchapter7x) [reply pkg/metaconfig/metaconfig.go]: agreed. i was struggling with this name.

**Comment by xchapter7x** [comment]
@Blokje5 
Change is in... ready to merge whenever you are :)

**Review by Blokje5 — APPROVED** [review]
LGTM!

**Pull request merged**